File: urna-eletronica-api/src/HubCount.Urna.Core/Models/Entities/Vote.cs
namespace HubCount.Urna.Core.Models.Entities
{
    public class Vote
    {
        public int Id { get; set; }
        public int? CandidateId { get; set; }
        public int TypeVote { get; set; }
        public DateTime DtVote { get; set; }
        public Candidate? Candidate { get; set; }
    }
}

File: urna-eletronica-angular/src/app/models/Vote.ts
export class Vote {
  constructor(candidateId: number, typeVote: number){
    this.candidateId = candidateId,
    this.typeVote = typeVote
  }

  id!: number;
  candidateId: number;
  dtVote!: Date;
  typeVote!: number;
}
